src/Events.Functions/Startup.cs

```diff
@@ -40,6 +40,7 @@ public void Configure(IWebJobsBuilder builder)
             builder.Services.AddSingleton<IQueueProcessorFactory, CustomQueueProcessorFactory>();
             builder.Services.AddSingleton<ITelemetryInitializer, TelemetryInitializer>();
             builder.Services.AddSingleton<IAccessTokenGenerator, AccessTokenGenerator>();
+            builder.Services.AddSingleton<ICertificateResolverService, CertificateResolverService>();
```

Comment:
The new Settings class must also be registered to Services/Configuration so that it is possible to override the default value through configuration.